### PR TITLE
snap: added tasks subcommand

### PR DESCRIPTION
**Pull request by stolowski**
Added 'tasks' subcommand to snap (works as 'change') and made 'change' hidden.

### CODE REVIEW

**Comment by stolowski** [comment]
@zyga I found only one occurence of such hint, it's when you type 'snap changes <digits>"; I fixed that. I haven't found any others, but grepping can be deceiving; let me know if you can think of any other use case where we would show it.

File: cmd/snap/cmd_changes.go [hunk 1]
```diff
@@ -46,13 +46,20 @@ type cmdChanges struct {
...
+
+type cmdTasks struct {
+	Positional struct {
 		ID changeID `positional-arg-name:"<id>" required:"yes"`
 	} `positional-args:"yes"`
 }
 
 func init() {
 	addCommand("changes", shortChangesHelp, longChangesHelp, func() flags.Commander { return &cmdChanges{} }, nil, nil)
 	addCommand("change", shortChangeHelp, longChangeHelp, func() flags.Commander { return &cmdChange{} }, nil, nil)
```
Comment (chipaca) [inline cmd/snap/cmd_changes.go]: you want to add `.hidden = true` to the end of that (see e.g. `cmd_booted.go`).

File: cmd/snap/cmd_changes.go [hunk 2]
```diff
@@ -46,13 +46,20 @@ type cmdChanges struct {
 
 type cmdChange struct {
 	Positional struct {
+		ID changeID `positional-arg-name:"<id>" required:"yes" hidden:"true"`
```
Comment (chipaca) [inline cmd/snap/cmd_changes.go]: that hidden hides the argument, but not the command

File: cmd/snap/cmd_changes.go (hunk 1 above)
Comment (stolowski) [inline cmd/snap/cmd_changes.go]: Thanks for spotting, fixed.

**Pull request merged**